python/lsst/daf/butler/configs/storageClasses.yaml

```diff
@@ -409,6 +409,9 @@ storageClasses:
     pytype: spectractor.fit.fitter.FitParameters
   ScarletModelData:
     pytype: lsst.scarlet.lite.io.ScarletModelData
+    parameters:
+      - blend_id
```

Comment:
One issue is going to be that if someone asks for a blend ID from an old JsonFormatter dataset it's going to break. I'm not sure if there is anyway to fix that and I don't know how important it is for it to work.

Reply:
Yes, I think that it's a good idea to include a comment in the (not yet written) deblending notebooks for DP1 that this feature will be available in DR1, and give an example of the API, but note that it will not work with DP1.